TaskLayer/MetaMorpheusTask.cs

```diff
@@ -343,21 +341,25 @@ protected static void WritePsmsToTsv(IEnumerable<PeptideSpectralMatch> items, st
                 {
                     output.WriteLine(heh.ToString(ModstoWritePruned));
                 }
-            }
+           }
         }
-
-        protected static HashSet<IDigestionParams> GetListOfDistinctDigestionParams(ICommonParameters commonParameters, IEnumerable<ICommonParameters> enumerable)
+        protected static HashSet<DigestionParams> GetListOfDistinctDigestionParams(CommonParameters commonParameters, IEnumerable<CommonParameters> enumerable)
```

Comment:
does this method need to exist? isn't it the same as `HashSet<DigestionParams>(commonParams.Select(p => p.DigestionParams))`

Reply:
CommonParameters does not allow .Select method